pkg/status/status.go

```diff
@@ -306,7 +306,7 @@ func getEndpointsInfos() (map[string]interface{}, error) {
 func getRemoteConfigStatus() map[string]interface{} {
 	status := make(map[string]interface{})
 
-	if expvar.Get("remoteConfigStatus") != nil {
+	if config.Datadog.GetBool("remote_configuration.enabled") && expvar.Get("remoteConfigStatus") != nil {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏Let's use `IsRemoteConfigEnabled` to catch GovCloud